Fix important issues and start updating code base

TheLukaszNs opened the pull request:
Fast summary
===
This PR resolves #45, partially #47 and #48 (I hope it does, only @vibhavagarwal5 had 
problem with this 😮).

Code base update:
- [x] `<ResponsiveImage />` -> `<Image />`
- [x] Moved images from `img` to `assets/images`
- [x] ~~Updated `React.PropTypes` with `prop-types`~~ (this was actually done - we didn't use old `PropTypes` from **React**)
- [ ] Move all styles to `assets` (I don't know if we should move all styles from its' components to separated files and create index file to store them all?)
- [x] Made additional styles update which is not actually code base update 🤔

vibhavagarwal5 commented:
I was already on it dude.

TheLukaszNs commented:
Oh, I saw You were working on caching images, weren't you? I didn't touch this issue.

TheLukaszNs commented:
I see You done much more work than me. I'll close my PR 😃

vibhavagarwal5 commented:
Yeah i am working on it. But still that if u would have commented on it that u were taking it, i would not have spent time on it. Since that issue was opened was me, it was my duty to close it asap. 😄  Thats why i worked on it.

vibhavagarwal5 commented:
Thanks @TheLukaszNs

agentmilindu commented:
@TheLukaszNs @vibhavagarwal5 Hope you two can work jointly on this. Use https://gitter.im/CodeLanka/ez-net-app for fast discussions, I too will be there.

vibhavagarwal5 commented:
@agentmilindu I think this is resolved.

TheLukaszNs commented:
I am reopening this PR as of additional issues have been opened - I had to fix them and I had my code base already updated by **me**.
I'll push patches to `develop` in the next minutes.

vibhavagarwal5 commented:
Aren't these exactly the changes i have done?

TheLukaszNs commented:
Yeah it's very similar as of it couldn't be done more differently. But if you make a closer look you will notice that it's made in the other way.
<hr width="33%"/>
It also fixes #51

vibhavagarwal5 commented on assets/styles/index.js:
```diff
@@ -0,0 +1,13 @@
+import detailScreen from './detailScreen';
+import * as loaderStyle from './loaderStyle';
+
+export {
```
Actually u don't need to do this again.
Since in each of your individual style.js u have exported the function, this is a repetition.

TheLukaszNs replied:
I know, I could just use `../styles/...` import but I wanted the code to be available like this (for example):
```javascript
import { loaderStyle } from '../assets/styles'
```
and not like this (which you can still use 😄):
```javascript
import loaderStyle from '../assets/styles/loaderStyle'
```
It's up to you which one you want to use.

vibhavagarwal5 replied:
for  `import { loaderStyle } from '../assets/styles'` to be used, you do `export loaderStyle` instead of `export default loaderStyle`

TheLukaszNs replied:
Oh, I know what you mean (I'll fix it now)! For the duplication - styles were already done, the point was just to move them to separate files - that's why it all looks so similar.

agentmilindu commented:
Ummm, I'm bit confused here. I see three PRs which kinda looks similar. Can you two please guide me through?

vibhavagarwal5 commented:
@agentmilindu PR #52 is very much different from the other two. Its regarding the caching issue.
Whereas PR #49 and #50 is about the code updation ie. issue #47 . Since I submitted it first with major changes, and then shortly after @TheLukaszNs submitted a similar PR, its looks duplicate to mine.

TheLukaszNs commented:
@vibhavagarwal5 I've submitted this PR first (10 mins before you). Then I decided to close it, because You did nearly the same work as me, but - you are right here - you've created issue #47 (however you did not inform that you'll do it). Yesterday I reopened this issue, merged changes in mine fork and updated this PR.

---
@agentmilindu those PRs are nearly the same because we were working on the same problem, but I've also fixed #51 and did some styles update.

vibhavagarwal5 commented:
Is issue #51 really an issue?

agentmilindu commented:
Let me go through the PRs and see.

agentmilindu commented:
@nushydude hey, I need your help here to get these two PRs merged.
